blog/2022-08-15-connect-pit-to-stryker-dashboard.md

```diff
@@ -0,0 +1,73 @@
+---
+slug: connect-pit-to-stryker-dashboard
+title: 'Connect PIT to Stryker Dashboard'
+author: Maarten Mulders
+author_title: Stryker Team
+author_url: https://maarten.mulders.it/
+author_image_url: https://maarten.mulders.it/img/twitter-profile-202106.png
+tags: [stryker, pit, java, maven]
```

Comment:
```suggestion
tags: [stryker, pit, java, maven, dashboard]
```

Reply:
> If you want, you could also add a [canonical url to your website](https://docusaurus.io/docs/markdown-features/head-metadata)

Cool, I did not even know this existed. If I understand correctly, it lets you hint search engines as to which version of (very) similar pages is "the original" and which one(s) is/are "the derivatives"?

Could be useful, but I'm too unfamiliar with Docosaurus to figure out from the docs how to do that. I tried adding this to the front matter:

```
canonical: https://maarten.mulders.it/2022/07/mutation-testing-badge-with-pit-and-stryker-dashboard/
```

But it seems not to work as I'd expect: the browser inspector still shows

```
<link rel="canonical" href="https://stryker-mutator.io/blog/connect-pit-to-stryker-dashboard/" data-rh="true">
```

Reply:
In the markdown article (so below the frontmatter) if you add:

```html
<head>
  <link rel="canonical" href="https://maarten.mulders.it/2022/07/mutation-testing-badge-with-pit-and-stryker-dashboard/" />
</head>
```

That should add it from what I can see in the documentation

Reply:
Thanks... I don't think I would've found that!

Is my interpretation of "canonical URL" correct, by the way?

Reply:
> Is my interpretation of "canonical URL" correct, by the way?

I think so, yeah, with it a search engine might decide to place the canonical article higher in the search results than the 'copy'

Reply:
Thanks for those suggestions!